WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModelBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿using ReactiveUI;
+using System;
+using WalletWasabi.Gui.ViewModels;
+using System.Diagnostics.CodeAnalysis;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class WalletViewModelBase : ViewModelBase, IComparable<WalletViewModelBase>
```

Comment:
We have all the information required in Wallet. Do not set them from outside.

Reply:
done